src/server/utils.spec.ts

```
@@ -1,4 +1,56 @@
-import { validatePayload } from './utils';
+import {
+  safeJsonStringify,
+  escapeHtml,
+  validatePayload,
+} from './utils';
```

ESLint

[error] Replace `⏎··safeJsonStringify,⏎··escapeHtml,⏎··validatePayload,⏎` with `·safeJsonStringify,·escapeHtml,·validatePayload·`

(prettier/prettier)